guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md

```diff
@@ -0,0 +1,430 @@
+---
+group: mftf
+title: Mftf commands
+version: 2.3
+github_link: magento-functional-testing-framework/release-2/commands/robo.md
+functional_areas:
+ - Testing
+mftf-release: 2.3.0
+---
+
+_This topic was updated due to the {{page.mftf-release}} MFTF release._
+{: style="text-align: right"}
+
+## Usage
+
+### Format
+
+Assuming that you're working in the project root directory in your terminal where you have installed mftf as a composer dependency, run Robo commands using the following format:
+
+```bash
+vendor/bin/mftf command [options] [arguments]
+```
+  
+## Useful commands
+
+The following are the most popular commands used to interact with the MFTF. For more commands, see [Reference] or run in your terminal:
+
+
+#### Build the Project
+
+```bash
+mftf build:project
+```
+
+#### Generate all tests in PHP
+
+```bash
+mftf generate:tests
+```
+#### Generate the specified test(s) in PHP
+
+```bash
+mftf generate:tests testName01 testName02 testName03
+```
+
+#### Run and generate all tests marked with the @group tag `example`
+
+```bash
+mftf run:group example
+```
+
+#### Run and generate all tests
+
+The command runs all the generated tests.
+
+```bash
+mftf run:tests
+```
+
+## Reference
+
+### general
+
+#### `reset`
+
+##### Description:
+Clean any configuration files from the environment (not including .env), as well as any generated artifacts.  
+
+##### Usage:
+`reset [options]`
+
+##### Options:
+opt                | description|
+---|---|
+`--hard` | parameter to force reset of configuration files.
+
+  
+### build
+ 
+#### `build:project`     
+
+##### Description:
+Clone the example configuration files and build the Codeception project.
+
+### generate
+
+#### `generate:tests`
+
+##### Description:
+Generate PHP code from the tests defined in XML files. (Path is determined by constant `TESTS_MODULE_PATH`)
+
+##### Usage:
+`generate:tests [options] [--] [<name>]...`
+
+##### Arguments:
+*arg*                | description|
+|---|---|
+`name` | name(s) of specific tests to generate
+
+##### Options:
+opt                | description|
+|---|---|
+`--config=CONFIG`   | default, singleRun, or parallel [default: "default"]  
+`--force`           | force generation of tests regardless of Magento Instance Configuration  
```

Comment:
It is not clear what means *regardless of Magento Instance Configuration*

Reply:
What configuration parameters or files are ignored?

Reply:
We ask Magento for the module read order when we parse the modules in, this is a flag to ignore the merge order and just parse the entire filesystem. Make sense?

Reply:
not sure I understand what you mean with your second comment?

Reply:
__regardless__ is the same as __ignoring__
Just wanted to understand what exactly that flag forces to ignore.
Your explanation is clear, thank you.